tests/test_particleContainer.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import numpy as np
+import amrex
+
+#TODO : Tests struct of arrays
+
+@pytest.fixture()
+def particle_container(std_geometry, distmap, boxarr):
+    pc = amrex.ParticleContainer_1_1_2_1(std_geometry, distmap, boxarr)
+    return pc
```

Comment:
Maybe extend the example and actually add a few particles in the particle container? :)

Reply:
done